### PR TITLE
feat(@settings): Build layout for settings wallet

### PR DESCRIPTION
**Pull request by alaibe**
image

No data plugged yet

### CODE REVIEW

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml [hunk 1]
```diff
@@ -15,8 +19,89 @@ Column {
...
+        anchors.rightMargin: -Style.current.padding
+    }
+
     StatusSettingsLineButton {
         text: qsTr("Networks")
+        height: 64
         onClicked: goToNetworksView()
     }
+
+    Separator {
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Profile/views/wallet/MainView.qml:70]: here giving the separator a height equivalent to the gap in between component would be a better approach ?
from the design it seems that the separator should have a height of 17 ? then we remove the need for the empty item?

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml [hunk 2]
```diff
@@ -15,8 +19,89 @@ Column {
...
+        font.pixelSize: 28
+        color: Theme.palette.directColor1
+    }
+
+    Item {
+        height: Style.current.bigPadding
+        width: parent.width
+    }
+
+    StatusSettingsLineButton {
```
Comment (Khushboo-dev-cpp) [inline ui/app/AppLayouts/Profile/views/wallet/MainView.qml:36]: clicking on this open a sub menu and there is no way to go back, can we please add a back button or as per the design, maybe you are planning to do it in a subsequent task?

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/Profile/views/wallet/MainView.qml:70]: currently if you notice the highlight as well the overlap the separator and it shouldn't
be that way

**Review by Khushboo-dev-cpp — CHANGES_REQUESTED** [review]
Hey I think some design parts are missing.

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml (hunk 2 above)
Comment (alaibe) [reply ui/app/AppLayouts/Profile/views/wallet/MainView.qml:36]: Yes that will be in subsequent task, Here i am focusing on the layout without the data

**Review by anastasiyaig — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml (hunk 1 above)
Comment (alaibe) [reply ui/app/AppLayouts/Profile/views/wallet/MainView.qml:70]: Good call updated I think

**Review by Khushboo-dev-cpp — APPROVED** [review]

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml [hunk 3]
```diff
@@ -15,8 +19,89 @@ Column {
 
     signal goToNetworksView()
 
+
+    StatusBaseText {
+        id: titleText
+        text: qsTr("Wallet")
+        font.weight: Font.Bold
+        font.pixelSize: 28
```
Comment (osmaczko) [inline ui/app/AppLayouts/Profile/views/wallet/MainView.qml:27]: Hardcoded font sizes should be avoided -> `font.pixelSize: Style.current.primaryTextFontSize`

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml [hunk 4]
```diff
@@ -15,8 +19,89 @@ Column {
...
+
+    Item {
+        height: Style.current.bigPadding
+        width: parent.width
+    }
+
+    StatusBaseText {
+        id: accountsText
+        text: qsTr("Accounts")
+        font.pixelSize: 15
```
Comment (osmaczko) [inline ui/app/AppLayouts/Profile/views/wallet/MainView.qml:86]: Same here

**Review by osmaczko — COMMENTED** [review]
Looks good :)

**Pull request merged**